docs/05_best-practices/04_data-design-and-migration.md

```diff
@@ -28,9 +28,9 @@ To learn how to modify the structure using ABI variants read this [tutorial](../
 
 #### 2.3.1. Migration without downtime, but slower
 
-1. Create the new version of your multi index table alongside the old one;
+1. Create the new version of your multi-index table alongside the old one;
```

Comment:
multi-index table OR multi-index table structure?

Reply:
it is fine either way in this context